blazor/gantt-chart/columns.md

```diff
@@ -331,6 +331,78 @@ Format | Formatted value
 
 ![Blazor Gantt Chart with Date Format](images/blazor-gantt-chart-date-format.png)
 
+## Autofit columns using method
```

Comment:
Give intro about autoFit and then explain method support.
Refer https://helpej2.syncfusion.com/angular/documentation/treegrid/columns/columns#autofit-columns 

Ensure the contents are revieed in ChatGPT or use content from above link